hvpm.py

```diff
@@ -48,7 +53,7 @@ def main(unused_argv):
 
   if FLAGS.status:
     print "Serialno : %d\n" % mon.getSerialNumber();
-    mon.fillAllStatusPacket()
```

Comment:
So,
I accidentally committed this work-in-progress :)

I did add fillAllStatusPacket (in only HVPM) in another pull request, but I had some questions as to what was the intention of it, or if my additions just belonged in fillStatusPacket.